### PR TITLE
Add MP OpenAPI 1.1 TCK project, mark feature for beta

### PR DESCRIPTION
**Pull request by arturdzm**

### CODE REVIEW

**Comment by arturdzm** [comment]
#build

**Comment by arturdzm** [comment]
#build

File: dev/com.ibm.ws.microprofile.openapi.1.1_fat_tck/publish/tckRunner/pom.xml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        http://www.eclipse.org/legal/epl-v10.html 
+    Contributors: 
+        IBM Corporation - initial API and implementation
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.ibm.ws.microprofile.openapi</groupId>
+    <artifactId>tck.runner</artifactId>
```
Comment (tevans78) [inline dev/com.ibm.ws.microprofile.openapi.1.1_fat_tck/publish/tckRunner/pom.xml]: You may wish to update this to be distinct from the openapi 1.0 TCK runner

Comment (arturdzm) [reply dev/com.ibm.ws.microprofile.openapi.1.1_fat_tck/publish/tckRunner/pom.xml]: Thanks fore suggestion, I will update it it next PR

**Pull request merged**